### PR TITLE
Add insurance details routing

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
See subject

https://github.com/HedvigInsurance/racoon/assets/6661511/33047159-525d-448b-9bf8-552f38f06e65

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3355** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3355" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3355?utm_source=stack-comment).

File: apps/store/src/features/memberArea/InsuranceSection/InsurancesSection.tsx [hunk 1]
```diff
@@ -52,6 +42,6 @@ const Grid = styled.div({
   },
 })
 
-export const InuranceButton = styled.button({
+export const InuranceLink = styled(Link)({
   cursor: 'pointer',
```
Comment (alebedev) [inline apps/store/src/features/memberArea/InsuranceSection/InsurancesSection.tsx:46]: Do we still need cursor style for Link?

**Review by alebedev — APPROVED** [review]
Looks good!

**Comment by gustaveen** [comment]
### Merge Activity

* **Oct 6, 6:04 AM**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3355).

**Pull request merged**